docs/api-docs/apps/tutorials/sample-app-next/05-step-3-integrate-api.md

```diff
@@ -0,0 +1,920 @@
+[Step 2: Connect your app to BigCommerce](../docs/Step-2-Connect-your-app-to-BigCommerce.md)
+
+# Step 3: Integrate the BigCommerce API and add a database
+
+After you have embedded your app in the BigCommerce platform, the next step is to integrate the BigCommerce API.
+
+Anytime you make an API call to BigCommerce, you need to pass in the access token. Storing the access token in a database, will allow you to persist the session when you call `/auth`, `/load`, or `/uninstall` endpoints. 
+
+In the purview of this tutorial, we will demonstrate how to integrate your app with [Cloud Firestore](https://firebase.google.com/docs/firestore/quickstart), a cloud-hosted NoSQLFirebase database, and [MySQL](https://www.mysql.com/), a relational database management system.
+
+## Install npm packages
+
+1. Install firebase, jsonwebtoken, mysql, and swr npm packages.
```

Comment:
we might want to only include either firebase or mysql here, depending on which one they prefer.  i.e. if you prefer Firebase, you'll also want to npm install firebase.  If you use sql, run `npm install --save mysql`